x/gamm/pool-models/stableswap/pool_test.go

```diff
@@ -166,24 +185,24 @@ func TestScaledSortedPoolReserves(t *testing.T) {
 			poolAssets:     fiveUnevenStablePoolAssets,
 			scalingFactors: []uint64{1, 1, 1, 1, 1},
 			expReserves: []osmomath.BigDec{
-				baseEvenAmt.MulInt64(3), // {"asset/c", baseEvenAmt.MulInt64(3)},
-				baseEvenAmt.MulInt64(4), // {"asset/d", baseEvenAmt.MulInt64(4)},
-				baseEvenAmt,             // {"asset/a", baseEvenAmt},
-				baseEvenAmt.MulInt64(2), // {"asset/b", baseEvenAmt.MulInt64(2)},
+				baseEvenAmt.MulInt64(3),
+				baseEvenAmt.MulInt64(4),
+				baseEvenAmt,
+				baseEvenAmt.MulInt64(2),
 				baseEvenAmt.MulInt64(5),
-			}, // {"asset/e", baseEvenAmt.MulInt64(5)}},
```

Comment:
Note: I removed these comments since `make format` was formatting them in an odd way (shuffling them, putting last one on newline etc.)